flink-table/flink-table-runtime-blink/pom.xml

```diff
@@ -74,6 +74,67 @@ under the License.
 			<version>${janino.version}</version>
 		</dependency>
 
+		<!-- Use TimestampString to cover CAST TIMESTAMP to STRING -->
+		<dependency>
+			<groupId>org.apache.calcite</groupId>
+			<artifactId>calcite-core</artifactId>
```

Comment:
I would prefer to not introduce calcite dependency for `table-runtime` module

Reply:
Is there any harmful things to introduce calcite dependency? 
The calcite's TimestampString provides standard-compliant processing of Timestamp literal which we can depend on.

Reply:
I recalled depending on calcite in different places introduced some packaging problems during 1.9.

Reply:
+1 to not introduce calcite to runtime.